test/util/test_path.cpp

```diff
@@ -161,7 +161,9 @@ bool test_dir_is_exfat()
     REALM_ASSERT_RELEASE(ret == 0);
     // The documentation and headers helpfully don't list any of the values of
     // f_type or provide constants for them
-    return fsbuf.f_type == 28 /* exFAT */;
```

Comment:
28 no longer works for me. Hopefully searching the f_fstypename will be more robust across time.